pkg/entity/project/project.go

```diff
@@ -119,6 +132,29 @@ func manage(args []string, cmd *cobra.Command) {
 	return
 }
 
+func get(projectName string) ProjectsWithActive {
+	projectId := GetProjectIdFromName(projectName)
+	req := &projects.GetProjectRequest{ProjectId: projectId}
+
+	response, err := client.GetProject(apiContext, req)
+	common.CliExit(err)
+
+	return ProjectsWithActive{
+		Projects:      []*entities.Project{response.Project},
+		activeProject: common.GetActiveProject(),
+	}
+}
+
+func del(projectName string) {
+	projectId := GetProjectIdFromName(projectName)
+	req := &projects.DeleteProjectRequest{
+		ProjectId: projectId,
+	}
+	_, err := client.DeleteProject(apiContext, req)
+	common.CliExit(err)
+	return
+}
+
 type ProjectsWithActive struct {
```

Comment:
Could you rename this struct to `ProjectsAndActiveProject`? I'm confused every time I read this struct's name.